README.md

```diff
@@ -4,6 +4,7 @@
 [![arXiv.org](https://img.shields.io/badge/arXiv-1109.0887-b31b1b.svg)](https://arxiv.org/abs/1109.0887)
 [![Python Versions](https://img.shields.io/pypi/pyversions/rgf_python.svg)](https://pypi.org/project/rgf_python)
 [![PyPI Version](https://badge.fury.io/py/rgf_python.svg)](https://badge.fury.io/py/rgf_python)
+[![Downloads](https://pepy.tech/badge/rgf-python)](https://pepy.tech/project/rgf-python)
```

Comment:
Please remove this badge from the general README. Let's leave this badge only in `rgf_python` README.

We have the same badge for R-package and it's presented only in R-package README. Due to their identical appearance it's confusing to have one of they or both of them on the main page.